packages/financial-templates-lib/src/helpers/allowances.js

```diff
@@ -0,0 +1,26 @@
+const { MAX_UINT_VAL } = require("@uma/common");
+const { getAbi } = require("@uma/core");
+
+// Sets `owner` allowance for `spender` to MAX, unless `spender` already has an allowance > 1/2 of the MAX_UINT.
+// Return array of successful approval transactions or undefined for skipped approvals.
```

Comment:
this comment implies this can do multiple approvals in one go. Wont it only do one at a time?

Reply:
Yup you're right, will fix comment